sage-ios/SAGE/SAGE/PastSemestersViewController.swift

```diff
@@ -0,0 +1,102 @@
+    var semesters: [Semester]?
+    var previousVC: UIViewController?
+    
+    override func viewWillAppear(animated: Bool) {
+        self.navigationController?.navigationBar.barTintColor = UIColor.lightGrayColor
+    }
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        self.title = "Past Semesters"
+        self.tableView.tableFooterView = UIView()
+        let n: Int! = self.navigationController?.viewControllers.count
+        self.previousVC = self.navigationController!.viewControllers[n-2]
+        
+        self.view.addSubview(self.activityIndicator)
+        self.activityIndicator.startAnimating()
+        
+        self.refreshControl = UIRefreshControl()
+        self.refreshControl?.backgroundColor = UIColor.lightGrayColor
+        self.refreshControl?.tintColor = UIColor.whiteColor()
+        self.refreshControl?.addTarget(self, action: "loadSemesters", forControlEvents: .ValueChanged)
+        
+        self.loadSemesters()
+    }
+    
+    override func viewWillLayoutSubviews() {
+        super.viewWillLayoutSubviews()
+        self.activityIndicator.centerInSuperview()
+    }
+    
+    func loadSemesters() {
+        SemesterOperations.loadSemesters({ (semestersArray) -> Void in
+            self.semesters = semestersArray
+            self.tableView.reloadData()
+            self.activityIndicator.stopAnimating()
+            self.refreshControl?.endRefreshing()
+            }) { (errorMessage) -> Void in
+                self.activityIndicator.stopAnimating()
+                self.refreshControl?.endRefreshing()
+                self.showErrorAndSetMessage(errorMessage)
+        }
+    }
+    
+    func showErrorAndSetMessage(message: String) {
+        let error = self.currentErrorMessage
+        let errorView = super.showError(message, currentError: error, color: UIColor.mainColor)
+        self.currentErrorMessage = errorView
+    }
+    
+    override func tableView(tableView: UITableView, heightForRowAtIndexPath indexPath: NSIndexPath) -> CGFloat {
+        return 44.0
+    }
+    
+    override func numberOfSectionsInTableView(tableView: UITableView) -> Int {
+        return 1
+    }
+    
+    override func tableView(tableView: UITableView, numberOfRowsInSection section: Int) -> Int {
+        if let semesters = self.semesters {
+            return semesters.count
+        }
+        return 0
+    }
+    
+    override func tableView(tableView: UITableView, cellForRowAtIndexPath indexPath: NSIndexPath) -> UITableViewCell {
+        var cell = tableView.dequeueReusableCellWithIdentifier("Semester")
+        if (cell == nil) {
+            cell = UITableViewCell(style: UITableViewCellStyle.Default, reuseIdentifier: "Semester")
+        }
+        cell!.selectionStyle = .None
+        cell!.textLabel!.text = self.semesters![indexPath.row].displayText()
+        cell!.textLabel!.font = UIFont.normalFont
```

Comment:
you can put this font and selection style lines (line 84) right when you init the cell under line 82